### PR TITLE
[Documentation] Query Param based Authentication

### PR DESCRIPTION
**Pull request by genego-io**

### CODE REVIEW

File: docs/web/api-v3.md [hunk 1]
```diff
@@ -145,6 +145,24 @@ response = requests.get(url, headers=headers)
...
+party service (e.g. Google sheets), without the option to specify the headers. In the below example, we have already
+generated the token through the steps above. The token is then used with the query parameter `token`
+
+```html
+http://app.konfuzio.com/api/v3/projects/3/export/all.csv?token=bf20d992c0960876157b53745cdd86fad95e6ff4
+```
+
+.. Note:: 
+   This functionality currently is only offered for
+   the [Google-Sheet export](https://help.konfuzio.com/integrations/google-sheet/index.html) of project Documents
```
Comment (atraining) [inline docs/web/api-v3.md:161]: please capitalize concepts, Project

File: docs/web/api-v3.md [hunk 2]
```diff
@@ -145,6 +145,24 @@ response = requests.get(url, headers=headers)
...
 
+#### Query based parameter token authentication
+
+For some endpoints we offer the ability to Authenticate with not just the token in the Header, but also by using the
+token in the request as a query parameter. This is particularly useful when needing to expose an endpoint to a third
+party service (e.g. Google sheets), without the option to specify the headers. In the below example, we have already
+generated the token through the steps above. The token is then used with the query parameter `token`
+
+```html
+http://app.konfuzio.com/api/v3/projects/3/export/all.csv?token=bf20d992c0960876157b53745cdd86fad95e6ff4
```
Comment (atraining) [inline docs/web/api-v3.md:156]: is the token real? can we display it in the public?

Comment (genego-io) [reply docs/web/api-v3.md:156]: Yes this can be displayed. Token is not real, and would not authenticate.

Comment (atraining) [reply docs/web/api-v3.md:156]: ok, you are only referring to Drive integration

what do you thinkg, can we add this to help.konfuzio.com that it is a second option for excel?

https://medium.com/lets-excel/ms-excel-use-powerquery-to-import-csv-formatted-data-from-the-web-a89d97893b04

Comment (genego-io) [reply docs/web/api-v3.md:156]: Yes we can. At least token based auth will remain the same once this has been implemented, and would work the same way for both excel and drive.

**Comment by atraining** [comment]
@genego-io why is this not merged?

**Pull request closed without merging**